Fix WebUI API doesn't update IP and Port

myfingerhurt opened the pull request:
Fix WebUI API doesn't update node address and port when adding the certificate, especially for hidden note once address changed, will never be able to get connected again.

Add a check about don't add own gpg_id.

Add an inactive friends list at last seen Unix timestamp API.

Add an remove a specific peer API.

myfingerhurt committed:
Fix WebUI doesn't update node address and port when add ceritificate, especially for hidden note once address changed, will never be able to get connected again

myfingerhurt committed:
Fix WebUI doesn't update node address and port when add ceritificate, especially for hidden note once address changed, will never be able to get connected again

csoler commented:
Thx for the contribution. However, I cannot review this PR since the diff is over the entire files. Please re-do your PR by limiting your changes to the actual changes in the code. Try not to re-indent the whole files in order to limit the scope of the changes.

myfingerhurt committed:
Change line by line in github web editor

Fix WebUI doesn't update node address and port when adding certificate, especially for hidden note once address changed, will never be able to get connected again

change few rsPeers to mRsPeers, probably copy and paste mistake?

myfingerhurt committed:
Change line by line in github web editor

Add two method handling list of inactive peers, and removing a specific node.

myfingerhurt commented:
Sorry, I have tried, as you can see, I copied the original version and replace my fixes, then based on it, editing line by line in the GitHub web editor, somehow the GitHub still thinking I have changed the whole file......

myfingerhurt commented:
I will close it and try again.
